monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -241,6 +255,19 @@ def generate_flight_grid_and_path_points(self, altitude_of_ground_level_wgs_84 :
 
         self.grid_cells_flight_tracks = all_grid_cell_tracks
 
+    def generate_flight_operator_details(self): 
+        ''' This class generates details of flights and operator details for a flight, this data is required for identifying flight, operator and operation  
+            Args:
+            locale of the format ISO 639 Language Code and - or _ followed by ISO 3166 Country code, not all locales are supported by the Faker module, '''
+        
+        my_flight_details_generator = details_generator.OperatorFlightDataGenerator()
+        
+        flight_details = { "serial_number":my_flight_details_generator.generate_serial_number(),"registration_number":my_flight_details_generator.generate_registration_number(),"operation_description":my_flight_details_generator.generate_operation_description()}
+        
+        operator_details =  { "name":my_flight_details_generator.generate_company_name(), "location":my_flight_details_generator.generate_operator_location(centroid= self.bbox_center[0])}
+
+        flight_operator_details = {'flight_details':flight_details, 'operator_details':operator_details}
```

Comment:
Dicts with arbitrary keys [make me nervous](https://github.com/interuss/dss/pull/488#discussion_r629671103); could we explicitly define this implicit type and also annotate that the `generate_flight_operator_details` method returns an instance of that type?